Documentation meta-descriptions review N° 4 (#5558)
wazuh/wazuh-documentation: source/user-manual/reference/tools/agent-control.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::

>   :description: Learn how the agent_control program allows you to query the manager for information about any agent and initiate a syscheck/rootcheck scan on an agent. 

Reviewer: ```suggestion
  :description: Learn to query the manager for information about any agent or initiate a syscheck/rootcheck scan on an agent using the agent_control program. 
```
